src/pages/workspaces/List.js

```diff
@@ -311,9 +312,9 @@ export const WorkspaceList = () => {
             const { workspace: { cloudPlatform } } = sortedWorkspaces[rowIndex]
             return div({ style: { ...styles.tableCellContainer, paddingRight: 0 } }, [
               div({ style: styles.tableCellContent }, [
-                Utils.switchCase(cloudPlatform,
-                  ['Gcp', () => h(CloudGcpLogo, { title: 'Google Cloud Platform', role: 'img' })],
-                  ['Azure', () => h(CloudAzureLogo, { title: 'Microsoft Azure', role: 'img' })])
+                Utils.switchCase(_.toUpper(cloudPlatform),
```

Comment:
Unfortunately the case of what we return in workspaces does not match what we do for billing projects and cloud environments. :(